apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -97,8 +98,16 @@ def create(
             All named columns must exist in the schema, and at least one
             index column name is required.
 
-        :param platform_config: Platform-specific options used to create this DataFrame,
-            provided via ``{"tiledb": {"create": ...}}`` nested keys.
+        :param domains: An optional sequence of min-max tuples specifying the
```

Comment:
is the "min-max tuple" inclusive or exclusive of the values specified?  I.e., open, closed or half-open ranges?

Reply:
updated